backend/src/main/java/com/sep6/backend/controller/AccountsController.java

```diff
@@ -1,18 +1,27 @@
 package com.sep6.backend.controller;
 
+import com.sep6.backend.models.Account;
 import com.sep6.backend.service.AccountsService;
+import lombok.AllArgsConstructor;
+import lombok.RequiredArgsConstructor;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.Mapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("/accounts")
+@AllArgsConstructor
 public class AccountsController {
 
     private AccountsService service;
-    @Autowired
-    public AccountsController(AccountsService service) {
-        this.service = service;
+
+    @PostMapping(value = "{id}")
```

Comment:
Should we use PUT request in the edit method? The PUT is currently used for other edit methods

Reply:
Usually put is used when we send just few fields and not the whole object
But I will change if if already all of the rest use put